openfl-tutorials/interactive_api/PyTorch_MedMNIST_2D/README.md

```diff
@@ -0,0 +1,64 @@
+# MedMNIST 2D Classification Tutorial
+
+![MedMNISTv2_overview](https://raw.githubusercontent.com/MedMNIST/MedMNIST/main/assets/medmnistv2.jpg)
+
+For more details, please refer to the original paper:
+**MedMNIST v2: A Large-Scale Lightweight Benchmark for 2D and 3D Biomedical Image Classification** ([arXiv](https://arxiv.org/abs/2110.14795))
+
+
+### 1. MedMNIST Installation and Requirements
+Setup the required environments and install `medmnist` as a standard Python package from [PyPI](https://pypi.org/project/medmnist/):
```

Comment:
Since you have a requirements file now, specify in the beginning that users can pip install -r requirements.txt file for installing all dependencies.